File: data/CodeLlama/readable_format_HPEC/fortran/file093.f90
program DRB039_truedepsingleelement_orig_yes
    use omp_lib
    implicit none

    integer :: len, i
    integer, dimension(:), allocatable :: a

    len = 1000
    allocate (a(len))

    a(1) = 2

    !$omp parallel do
    do i = 1, len
        a(i) = a(i)+a(1)
    end do
    !$omp end parallel do

    print 100, a(500)
    100 format ('a(500) =',i3)

    deallocate(a)
end program

File: data/llama3_8B_ins_results/readable_format_HPEC/cpp/file093.cpp
#include <iostream>
#include <vector>
#include <omp.h>

int main() {
    int len = 1000;
    std::vector<int> a(len);

    a[0] = 2;

    #pragma omp parallel for
    for (int i = 0; i < len; i++) {
        a[i] += a[0];
    }

    std::cout << "a(500) = " << a[500] << std::endl;

    return 0;
}
